pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -688,25 +710,37 @@ const storeSyslogTemplate = `{{- define "storeSyslog" -}}
   keep_alive_cnt 9
   keep_alive_intvl 7200
 {{ end -}}
+
 	{{if .PayloadKey -}}
 	<format>
 	  @type single_value
 	  message_key {{.PayloadKey}}
 	</format>
 	{{end -}}
```

Comment:
non-blocker but while you're fixing `<buffer>` indentation can you also fix the indentation in the rest of this `<store>`? it can be an additional commit if you want..